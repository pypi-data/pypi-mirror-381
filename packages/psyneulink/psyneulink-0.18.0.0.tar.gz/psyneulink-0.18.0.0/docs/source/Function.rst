Function
========

.. toctree::
   :maxdepth: 1

   NonStatefulFunctions
   StatefulFunctions
   UserDefinedFunction

.. toctree::
   :maxdepth: 3

.. automodule:: psyneulink.core.components.functions.function
   :members: Function_Base, ArgumentTherapy, RandomMatrix
   :private-members:
   :exclude-members: Parameters
