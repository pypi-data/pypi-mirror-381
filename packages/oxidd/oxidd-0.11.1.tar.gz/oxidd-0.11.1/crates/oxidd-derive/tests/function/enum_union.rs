use oxidd_derive::Function;

#[derive(Function)]
enum Foo {}

#[derive(Function)]
union Bar {
    x: u32,
    y: bool,
}

fn main() {}
