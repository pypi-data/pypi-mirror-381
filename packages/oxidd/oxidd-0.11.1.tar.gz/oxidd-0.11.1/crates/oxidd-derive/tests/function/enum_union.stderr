error: `Function` can only be derived for `struct`s
 --> tests/function/enum_union.rs:4:1
  |
4 | enum Foo {}
  | ^^^^

error: `Function` can only be derived for `struct`s
 --> tests/function/enum_union.rs:7:1
  |
7 | union Bar {
  | ^^^^^
